server/routes/user.ts

```diff
@@ -33,9 +34,10 @@ router.post("/register", userController.processRegisterPage);
 router.get("/logout", userController.processLogoutPage);
 
 //NEW GET account page through displaydisplayAccountPage method
-router.get("/account", userController.displayAccountPage);
+router.get("/account", util.authguard ,userController.displayAccountPage);
 
 /* Get - display ediuser page - with /edituser */
-router.get("/edituser", (req, res, _next) => {
+router.get("/edituser/:id", (req, res, _next) => {
```

Comment:
1. We need to add the authGuard here.
2. Use the route handler created in the `server/controllers/user.ts` file.